src/python/visclaw/data.py

```diff
@@ -621,12 +622,17 @@ def __init__(self, name, figno, fig_type, plotdata):
         self._show = True
         self.add_attribute('plotaxes_dict', {})
         self.add_attribute('type',fig_type)   # = 'each_frame' or 'each_run' or 'each_gauge'
+        self.add_attribute('use_for_kml',False)
+        self.add_attribute('kml_dpi',200)
+        self.add_attribute('kml_xlimits',None)
```

Comment:
I just tried plotting by relying only on the 'auto' selection mode, and half way through the simulation. Matplotlib changes its mind about what the axes limits should be.  Early figures match the GE latlong box (hardwired with kml_xlimts/ylimts), and later ones don't.     

So I guess one question is, should the GE box be allowed to  change during the simulation?  (I say no, at least in this version).